### ISSUE TITLE
Mark review assignment when it's opened by an editor

### ISSUE THREAD

**Issue by Vitaliy-1** [pkp/pkp-lib#9799]
After review is complete, editor is able to view review details. This event isn't reflected anywhere in the system.
`considered` review assignment property [holds 4 values](https://github.com/pkp/pkp-lib/blob/a95b221969aa35aa25540c23f8874a9226fcc49e/classes/submission/reviewAssignment/ReviewAssignment.php#L40-L43): `REVIEW_ASSIGNMENT_NEW`, `REVIEW_ASSIGNMENT_CONSIDERED`, `REVIEW_ASSIGNMENT_UNCONSIDERED`, `REVIEW_ASSIGNMENT_RECONSIDERED`. Add a new value to reflect that the review was viewed but not considered. This will help submission status tracking on the review stage.

pkp-lib: https://github.com/pkp/pkp-lib/pull/10233
tests OJS: https://github.com/pkp/ojs/pull/4682

**Comment by jardakotesovec** [comment]
@Vitaliy-1 Would that also introduce new review assignment status? `REVIEW_ASSIGNMENT_STATUS_READ` or `REVIEW_ASSIGNMENT_STATUS_VIEWED`, something like that?

**Comment by Vitaliy-1** [comment]
Yes, was thinking about `REVIEW_ASSIGNMENT_STATUS_VIEWED`. Another way could be just highlight on the new dashboard review assignments completed by the reviewer but without a a considered status from an editor for a certain amount of time.

**Comment by Vitaliy-1** [comment]
Filed a bug that is related: https://github.com/pkp/pkp-lib/issues/10232

**Comment by Vitaliy-1** [comment]
PR pkp-lib: https://github.com/pkp/pkp-lib/pull/10233

**Comment by Vitaliy-1** [comment]
@jardakotesovec, do we need also to record a date when review assignment was first opened by the editor?

**Comment by jardakotesovec** [comment]
@Vitaliy-1 From current UI plans that we have for review assignments popups - we don't need it. 

But what about review history? It would probably make sense to include it there, but maybe thats coming from different logging table?

<img width="851" alt="Screenshot 2024-08-06 at 15 40 12" src="https://github.com/user-attachments/assets/82be040b-f526-4e87-bdf4-4b8c1eb399e0">

**Comment by Vitaliy-1** [comment]
@jardakotesovec, looking into the actual code of the review assignment history - that information wasn't planned to be included there. I can add considered date after @Hafsa-Naeem finishes the work on this issue: https://github.com/pkp/pkp-lib/issues/10359

Also, why do you need `REVIEW_ASSIGNMENT_STATUS_VIEWED`? Technically it's not a status and better corresponds to the `REVIEW_ASSIGNMENT_STATUS_RECEIVED`. And for `REVIEW_ASSIGNMENT_CONSIDERED` and `REVIEW_ASSIGNMENT_RECONSIDERED` correspondent status is `REVIEW_ASSIGNMENT_STATUS_COMPLETE`

**Comment by jardakotesovec** [comment]
@Vitaliy-1 So far I have been using only review assignment status to differentiate between the states.

What we would like to achieve is to capture state between REVIEW_ASSIGNMENT_STATUS_RECEIVED and REVIEW_ASSIGNMENT_STATUS_COMPLETE, where we would indicate that the review assignment was read.

Technically I could additionally check the `considered` value, but it seems that the `considered` is just one of the inputs to the calculation of review assignment status? Thats why I expected this also result in new review assignment status.

So the status transitions would be REVIEW_ASSIGNMENT_STATUS_RECEIVED -> REVIEW_ASSIGNMENT_STATUS_VIEWED -> REVIEW_ASSIGNMENT_STATUS_COMPLETE

If you prefer I should be able use consider in addition to the status to determine it. Even though it seems it should be covered in schema-  I don't see `considered` property in review assignment in the submission listing. And oddly considered column has only NULL values..  which seems suspicious - I thought that confirmed review would have some value there.

**Comment by jardakotesovec** [comment]
Ops.. was looking to wrong database..  so the considered value in database is ok. But don't see it in the API

**Comment by Vitaliy-1** [comment]
Fixes:
pkp-lib stable 3.5.0: https://github.com/pkp/pkp-lib/pull/11166
tests: https://github.com/pkp/ojs/pull/4782

**Comment by Tribunal33** [comment]
I believe this is working.  @Vitaliy-1 I tested your remote branch and found the following DB changes being applied to the Review_assignments table, considered column.  Can merge now.  It flips back and forth from 1 to 2 when decision is reverted.  It does not go back to 4 (unread state).

Add a reviewer DB considered value = 0
Login as reviewer and complete the review DB considered value = 0
View Unread review DB considered  value = 4
Confirm review as editor DB considered value = 2
Revert decision DB considered value = 1
Reading review DB considered value = 1 (unchanged is expected)
Confirming review again as editor DB considered value = 2
